db/seeds.rb

```diff
@@ -1,3 +1,7 @@
+unless StudyCode.find_by(title: 'A1543457')
+  StudyCode.create!(title: 'A1543457')
+end
+
```

Comment:
This is the part which resolves https://github.com/Australian-Genomics/CTRL/issues/29. I know it should be its own PR, but it's just four teeny-tiny lines...